web/src/components/Plugin/PluginPage.tsx

```diff
@@ -180,16 +180,17 @@ const PluginPage: React.FC<Props> = ({
                   key={item.name}
                   actions={[
                     <Button
-                      type={
-                        initialData[item.name] && !initialData[item.name].disable
-                          ? 'primary'
-                          : 'default'
-                      }
+                      type='primary'
+                      danger= {initialData[item.name] && !initialData[item.name].disable}
```

Comment:
```suggestion
                      type= {initialData[item.name] && !initialData[item.name].disable ? 'primary' : 'default'}
                      danger= {initialData[item.name] && !initialData[item.name].disable}
```

Reply:
I think the button should be displayed as default before enabling, it is better to be primary after enabling.

Reply:
like this :
![image](https://user-images.githubusercontent.com/72343596/113429218-bc059b80-940a-11eb-81ec-cd0396c2109d.png)

Reply:
Update commit :)